src/main/python/main/ayab/transforms.py

```
@@ -96,14 +96,14 @@ def reflect(
         h1 = 1 + h0 + h_
         if w1 > 1:
             im = image
-            image = Transform.hflip(image)
-            image = Transform.repeat(image, (1, w1))
+            image = InageTransform.hflip(image)
```

Ruff (0.8.2)

Undefined name `InageTransform`

(F821)